lib/web_ui/test/golden_tests/engine/canvas_draw_image_golden_test.dart

```diff
@@ -23,7 +23,7 @@ void main() async {
   // Commit a recording canvas to a bitmap, and compare with the expected
   Future<void> _checkScreenshot(RecordingCanvas rc, String fileName,
       {Rect region = const Rect.fromLTWH(0, 0, 500, 500),
-        double maxDiffRatePercent = 0.0}) async {
+        double maxDiffRatePercent = 0.0, bool write = false}) async {
```

Comment:
remove write:false

Reply:
It's been useful the last few times I added new screenshot tests, but I can remove it if adds noise.